devices.js

```diff
@@ -12530,14 +12530,16 @@ const devices = [
         description: 'Universal wink enabled white ceiling fan premier remote control',
         fromZigbee: preset.light_onoff_brightness().fromZigbee.concat([fz.fan]),
         toZigbee: preset.light_onoff_brightness().toZigbee.concat([tz.fan_mode]),
-        exposes: [e.light_brightness(), e.fan()],
+        exposes: [e.light_brightness(), exposes.climate().withFanMode(['off', 'low', 'medium', 'high', 'on', 'auto', 'smart'])],
```

Comment:
I don't understand this change, could you explain it?

Reply:
The fact that it is not displayed correctly in the frontend is not a reason to change it here. Can you provide a screenshot of it here?

Reply:
before the change
![Capture d’écran 2021-04-01 145219](https://user-images.githubusercontent.com/23260814/113341850-b2414280-92fb-11eb-903a-9a923db5c028.png)

after the change
![Capture d’écran 2021-04-01 145805](https://user-images.githubusercontent.com/23260814/113341884-bc634100-92fb-11eb-8847-6e206c75d4ed.png)

I know the frontend is a different project, but it does seem to work better this way, so I figured it was expected to be this way. as I said before, it's a ceiling fan with a light, it would be right to have the fan part be the first function that is exposed, if you think it's better

Reply:
@nurikk can you look into this? Seems fan is not supported in the frontend

Reply:
Done